Example/ContactsListSettingsTableViewController.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+import MiniApp
+
+class ContactsListSettingsTableViewController: UITableViewController {
+
+    var userContactList: [String?: Any] = [:]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.separatorStyle = .singleLine
+        self.tableView.tableFooterView = UIView(frame: .zero)
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath) as UITableViewCell? ?? UITableViewCell()
+        cell.textLabel?.text = "User Contact - " + String(indexPath.row + 1)
```

Comment:
Can we make the label be the actual UUID instead? The UUID is the only thing that will be passed to the Mini App, so this label isn't useful to the user.

Reply:
Yes, we can. I thought displaying UUID would confuse the user. I will do it 👍

Reply:
Done